### PR TITLE
Remove the create/delete operator actions from the `epinio app chart` ensemble.

### LINKED ISSUE #1400: Remove the create/delete operator actions from the `epinio app chart` ensemble.
We currently expose the `epinio app chart create` and `... delete` commands.
These commands signify operator actions which should not be available to developers.
Hide the commands at least.
Better remove them entirely.

@agracey Comments ?

### PR DESCRIPTION
**Pull request by andreas-kupries**
For all that the issue is in the icebox, the base change looked to be trivial, so here it is, with some fixes to the cli doc generation.
The larger change, **TODO**, is updating the test suite, as it currently uses and tests the chart create/delete commands.

fix #1400 

fix: remove chart operator commands, minimal change
fix: typo in operator command create
fix: cli doc generation.

### CODE REVIEW

**Comment by andreas-kupries** [comment]
api failure looks to be imported from main - not in modified code/tests, was noted in slack for other branch, possibly even main.

File: Makefile
Comment (jimmykarily) [inline Makefile:122]: Is this correct? What is the old path used for?

Comment (andreas-kupries) [reply Makefile:122]: The `docs/src` became `docs/docs` in the move to docasaurus. The old path is gone.

Comment (andreas-kupries) [reply Makefile:122]: So, IMHO this change is correct.

File: scripts/cli-docs-generate.sh
Comment (jimmykarily) [inline scripts/cli-docs-generate.sh:34]:
```suggestion
```

**Review by jimmykarily — APPROVED** [review]

File: scripts/cli-docs-generate.sh
Comment (andreas-kupries) [reply scripts/cli-docs-generate.sh]: Accepted and committed. @jimmykarily, this has dismissed your approval.

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**
